### DISCUSSION TITLE
Squad Search capability

### DISCUSSION THREAD

**Discussion by ZelalemGizachew** [dailydotdev/daily#853] [Feature Requests]
A squad searching feature for us to explore squads to join to. Categorizing the squad as public and private may address of the squad that don't want anyone to join with out a request.

**Comment by idoshamun** [comment]
Hi @ZelalemGizachew,

Public squads with a directory to explore new squads is definitely on our roadmap.
Stay tuned!

**Reply by ZelalemGizachew** [reply]
Ok Great!

**Reply by nimrodkra** [reply]
@ZelalemGizachew icymi https://app.daily.dev/posts/5IowGmxNA

![Screenshot 2023-08-22 at 17 50 37](https://github.com/dailydotdev/daily/assets/41470823/e6e95a80-1ca3-42ce-bea7-0d2bf99074ee)

**Comment by FifthWye** [comment]
I'm not sure if I misunderstood @ZelalemGizachew but when I was about to create almost the same topic I saw this one and the status "Completed". But that's how I understand that:

I would like to find some specific squads like _web crawling_ squads for example but It's not that easy to find such squads just by categories at the top 
![image](https://github.com/user-attachments/assets/b41fd1ad-0c5e-4ea3-a200-1be1d43a7036)

And when I try to use search to find it it only returns the results for tags and posts 
![image](https://github.com/user-attachments/assets/52614334-f5b2-4f05-84f5-d02c330a9d89)

So the question is if a search by squads was added and I just can't find it or the public squads page is an alternative to what that person asked for?

**Reply by nimrodkra** [reply]
@ZelalemGizachew the capability to find Squads through the search bar exists and live already. There is probably no Squad around crawling atm so feel free to create one :) 

![Screenshot 2024-11-03 at 14 01 01](https://github.com/user-attachments/assets/f2aa3952-eabc-4685-995a-f09b08aefb9a)

**Reply by FifthWye** [reply]
@nimrodkra ok, I guess there are just no squads for the topics I was searching for yet. Thanks for the explanation 👍

**Reply by ZelalemGizachew** [reply]
@nimrodkra Thank you for the replay but that wasn't my question.
